test/run_tests.py

```diff
@@ -476,6 +510,46 @@ def get_deps(test_number, tests):
 
     return list(tests_to_run_explicitly)
 
+def convert_tests_for_flatbuffers(tests, to_flatbuff, working_dir, color_enum):
+    test_base_working_dir = str(working_dir)
+    if not Path(test_base_working_dir).exists():
+        Path(test_base_working_dir).mkdir(parents=True, exist_ok=True)
+
+    for test in tests:
+        test_id = test['id']
+        if 'vw_command' not in test:
+            print("{}Skipping test {} for flatbuffers, no vw command available{}".format(color_enum.LIGHT_CYAN, test_id, color_enum.ENDC))
+            continue
+        if 'flatbuffer' in test['vw_command']:
+            print("{}Skipping test {} for flatbuffers, already a flatbuffer test{}".format(color_enum.LIGHT_CYAN, test_id, color_enum.ENDC))
+            continue
+        if 'malformed' in test['vw_command']:
+            print("{}Skipping test {} for flatbuffers, malformed input{}".format(color_enum.LIGHT_CYAN, test_id, color_enum.ENDC))
+            continue
+        if 'input_files' not in test:
+            print("{}Skipping test {} for flatbuffers, no input files{}".format(color_enum.LIGHT_CYAN, test_id, color_enum.ENDC))
+            continue
+        if 'dictionary' in test['vw_command']:
+            print("{}Skipping test {} for flatbuffers, currently dictionaries are not supported{}".format(color_enum.LIGHT_CYAN, test_id, color_enum.ENDC))
+            continue
+        if 'invert_hash' in test['vw_command']:
+            print("{}Skipping test {} for flatbuffers, invert_hash not supported on converted files{}".format(color_enum.LIGHT_CYAN, test_id, color_enum.ENDC))
+            continue
+        if 'audit' in test['vw_command']:
+            print("{}Skipping test {} for flatbuffers, audit not supported{}".format(color_enum.LIGHT_CYAN, test_id, color_enum.ENDC))
+            continue
+        if 'dsjson' in test['vw_command']:
```

Comment:
`cats` and `dsjson` will not need to be skipped once the 2 related PR's (linked in description) are merged